src/pypowsybl_jupyter/networkexplorer.py

```diff
@@ -69,6 +77,12 @@ def toggle_switch(event: any):
         update_sld_diagram(True)
         update_nad_diagram()
 
+    def go_to_vl_from_map(event: any):
+        vl_from_map= str(event.selected_vl)
+        set_current_vl(vl_from_map)
+        #switch to the SLD tab
```

Comment:
I realized that the switch might be really slow if the NAD-depth is high. This is not due to this PR (also occurs when navigating with arrows on SLD) so I'll open a new issue about that